frontend/ui/widget/virtualkeyboard.lua

```diff
@@ -760,6 +760,7 @@ local VirtualKeyboard = FocusManager:new{
         ko_KR = "ko_KR_keyboard",
         ru = "ru_keyboard",
         tr = "tr_keyboard",
+        vi = "vn_keyboard",
```

Comment:
Strange, Weblate doesn't use ISO 3166? But for simplicity's sake it's would be better to name it vi_keyboard instead of introducing multiple different designations for the same thing.